Scalar.Common/Platforms/Mac/MacPlatform.cs

```diff
@@ -148,7 +151,32 @@ public override void IsServiceInstalledAndRunning(string name, out bool installe
 
         public override string GetTemplateHooksDirectory()
         {
-            return Path.Combine("/usr", "local", "git", ScalarConstants.InstalledGit.HookTemplateDir);
+            string gitExecPath = GitInstallation.GetInstalledGitBinPath();
+
+            // Resolve symlinks
+            if (NativeMethods.IsSymlink(gitExecPath))
+            {
+                string link = NativeMethods.ReadSymlink(gitExecPath);
```

Comment:
Is there maybe any way to call the [`realpath()`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/realpath.html) function? That would most likely avoid having to implement the symlink-following logic.

Reply:
Yeah, unfortunately .NET has a [long outstanding proposal for dealing with symlinks](https://github.com/dotnet/runtime/issues/24271). :(

Reply:
Perhaps we consider [this StackOverflow question/answer](https://stackoverflow.com/a/33487494/127088):

```
var path = NativeMethods.GetFinalPathName(@"c:\link");
```

Reply:
This code is using the Win32 API.. the problem we have is on UNIX machines..

Reply:
Updated to use `realpath`!